cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/handler/async/runnable/RunnableUtils.java

```diff
@@ -354,11 +355,9 @@ public static void sanityCheckOfflineReplicaPresence(ClusterModel clusterModel)
     private final Set<Integer> _recentlyDemotedBrokers;
 
     public RecentBrokers(Set<Integer> recentlyRemovedBrokers, Set<Integer> recentlyDemotedBrokers) {
-      if (recentlyRemovedBrokers == null || recentlyDemotedBrokers == null) {
-        throw new IllegalArgumentException("Attempt to set a null value for recent brokers.");
-      }
-      _recentlyRemovedBrokers = recentlyRemovedBrokers;
-      _recentlyDemotedBrokers = recentlyDemotedBrokers;
+      String errorMsg = "Attempt to set a null value for recent brokers.";
```

Comment:
Instead of reusing the error message, I think we can be more specific by:

```
_recentlyRemovedBrokers = validateNotNull(recentlyRemovedBrokers, "Attempt to set a null value for recent removed brokers.");
_recentlyDemotedBrokers = validateNotNull(recentlyDemotedBrokers, "Attempt to set a null value for recent demoted brokers.");
```

Reply:
Thanks for your input. I've made the error message more specific.